app.json

```diff
@@ -181,12 +181,17 @@
     "TWILIO_VALIDATION": {
       "description": "Validate twilio message report links as well",
       "value": "1"
+    },
+    
+    "TWILIO_MESSAGE_SERVICE_SID": {
+      "description": "A twilio service message account SID",
+      "required": false
     }
   },
   "addons": [
-    "heroku-postgresql:hobby-dev",
+    "heroku-postgresql:mini",
     {
-      "plan": "heroku-redis:hobby-dev",
+      "plan": "heroku-redis:mini",
```

Comment:
If I remember right, we have documentation that provides estimated costs for running Spoke in Heroku. Since this looks like a different postgresql tier, should we be updated that documentation along with this change?

Reply:
That makes a lot of sense @lastminutediorama !  Yes, I will create a ticket to provide that update and deliver that to this branch.